src/Components/Navigation/Navigation.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Navigation.css';
```

Comment:
Since the other group made their styling changes in `App.css` which you deleted, the file will reappear after merging in the app again. So, maybe you could do your styling in App.css too.
(Before deleting something because you may have no need for it now, you should talk about deleting the file with the other group to avoid these kinds of conflicts).